extensions/amp-bind/0.1/test/test-bind-expression.js

```diff
@@ -240,6 +240,15 @@ describe('BindExpression', () => {
       expect(evaluate('"abc".toUpperCase()')).to.equal('ABC');
     });
 
+    it('escaped quotes', () => {
+      expect(evaluate(`'\\"'`)).to.equal(`'"'`);
```

Comment:
Shouldn't this equal ``` `"` ```?

Reply:
https://github.com/rg3/youtube-dl/blob/master/LICENSE